tests/testTriggers.go

```diff
@@ -2106,6 +2106,8 @@ func TriggerStorageFullPoolExpansion(contexts *[]*scheduler.Context, recordChan
 
 				}
 			}
+			log.InfoD("Printing The storage pool status after pool expansion")
```

Comment:
we have other tests adddisk and resizedisk. please add this in those tests also.
Also add this for tests in storage_pool_test.go also

Reply:
Added the log accordingly and sv pool show will now happen on the nodes on which storage pool operation happen